packages/cli/lib/validators/logValidatorResults.js

```diff
@@ -0,0 +1,51 @@
+const { logger } = require('@hubspot/cli-lib/logger');
+const { VALIDATION_RESULT } = require('./constants');
+
+function logResultsAsJson(results) {
+  let success = true;
+  const resultObj = results.reduce((acc, result) => {
+    if (!acc[result.validator]) {
+      if (success && result.result !== VALIDATION_RESULT.SUCCESS) {
+        success = false;
+      }
+      acc[result.validatorKey] = [];
+    }
+    const {
+      validatorName: __omit1, // eslint-disable-line no-unused-vars
+      validatorKey: __omit2, // eslint-disable-line no-unused-vars
+      ...resultWithOmittedValues
+    } = result;
+    acc[result.validatorKey].push(resultWithOmittedValues);
+    return acc;
```

Comment:
My Javascript is not the strongest so I apologize if I'm missing something obvious here. Where does `resultWithOmittedValues` come from and what does that data look like?

Reply:
I think I mostly explained where `resultWithOmittedValues` is coming from in my answer above, but I'll share what the shape of it looks like. All I'm doing here is omitting the `validatorKey` and `validatorName` fields from the object before we output the json because I don't think either of those fields are useful for the BE or any other consuming system. I'm only using them for error grouping and console logging purposes.
```js
// result
{
  validatorKey: 'templateLimit',
  validatorName: 'Template limit',
  key: 'templateLimit.limitExceeded',
  result: 'FATAL',
  error: 'blah blah blah'
}

// resultWithOmittedValues
{
  key: 'templateLimit.limitExceeded',
  result: 'FATAL',
  error: 'blah blah blah'
}
```